01_Core/week04/ecommerce/contact/index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+
+<html>
+	<head>
+		<meta charset="utf-8">
+		<meta name="viewport" content="width=device-width">
+		<title>Stardust Designs || Contemporary Furniture</title>
+		<link rel="stylesheet" href="../stylesheets/normalize.css">
+		<link rel="stylesheet" href="../stylesheets/style.css">
+		<script src="validation.js"></script>
+	</head><!--/head-->
```

Comment:
I love the commenting, but since the closing tag is self-explanatory this is unnecessary.